hibernate-query-interpreter/src/main/antlr/org/hibernate/query/parser/internal/hql/antlr/HqlParser.g4

```diff
@@ -181,7 +185,7 @@ dynamicInstantiationArgExpression
 	;
 
 jpaSelectObjectSyntax
-	:	objectKeyword LEFT_PAREN IDENTIFIER RIGHT_PAREN
+	:	objectKeyword LEFT_PAREN alias RIGHT_PAREN
```

Comment:
The identifier here is a reference to a from-clause alias, not declaring an alias.